src/components/expanded-state/TokenIndexExpandedState.js

```diff
@@ -180,15 +182,17 @@ export default function TokenIndexExpandedState({ asset }) {
         ) : (
           <SheetActionButtonRow>
             <Column marginTop={5}>
-              <SwapActionButton
-                asset={assetWithPrice}
-                color={color}
-                inputType={AssetInputTypes.out}
-                label={`􀖅 ${lang.t('expanded_state.token_index.get_token', {
-                  assetSymbol: asset?.symbol,
-                })}`}
-                weight="heavy"
-              />
+              {!isTestnetNetwork(network) && (
```

Comment:
very very small thing, could you leave it like you did here https://github.com/rainbow-me/rainbow/pull/3795/files#diff-a6526b1e406531996c65cf88b91d54c5e11cfbb567bb143e6ae61126f427a206R212  in a const outside